routes/v1.js

```diff
@@ -1040,8 +1056,10 @@ api.declare({
 
   // Authenticate request by providing parameters
   if (!req.satisfies({
+    taskId,
+    runId,
     workerGroup:    run.workerGroup,
-    workerId:       run.workerId
+    workerId:       run.workerId,
```

Comment:
Ah, wait to stay on top of the latest trailing comma craze :)  I just didn't know because you added a couple of things to the end of an object (line 1009) and didn't have a trailing comma :)